MultiServer.py

```diff
@@ -763,30 +763,25 @@ async def on_client_connected(ctx: Context, client: Client):
         for slot, connected_clients in clients.items():
             if connected_clients:
                 name = ctx.player_names[team, slot]
-                players.append(
-                    NetworkPlayer(team, slot,
-                                  ctx.name_aliases.get((team, slot), name), name)
-                )
+                players.append(NetworkPlayer(team, slot, ctx.name_aliases.get((team, slot), name), name))
     games = {ctx.games[x] for x in range(1, len(ctx.games) + 1)}
     games.add("Archipelago")
     await ctx.send_msgs(client, [{
-        'cmd': 'RoomInfo',
-        'password': bool(ctx.password),
-        'games': games,
+        "cmd": "RoomInfo",
```

Comment:
For a large, possibly-breaking pull request, I think it would be valuable to put small, non-breaking style changes like this in a separate PR, so that they don't distract from things that need to be looked at in this PR.

Reply:
I tried to limit any style-guideline updates to any sections I needed to tweak. The amount of these additional style changes shouldn't be a heavy cognitive load compared to the rest of the PR.

Reply:
I agree with beauxq here. It is impossible to see what changed in this code block if anything changed at all.

And `datapackage_checksums` still using single quotes completely negates any positive effect this would have.

Reply:
Ultimately, I still disagree as it seems clear in GitHub UI to me, but I reverted the changes regardless.